### PR TITLE
Development

### PR DESCRIPTION
**Pull request by brquispe**

### CODE REVIEW

File: src/Products/components/ProductCard/ProductCard.module.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
+.productCard {
+  background-color: #FFF;
+  border-radius: 5px;
+  color: var(--color-third);
+  font-family: 'Montserrat';
+  display: flex;
+  flex-flow: column wrap;
+  gap: .5rem;
+  box-shadow: rgba(0, 0, 0, 0.15) 1.95px 1.95px 2.6px;
+  /* max-width: 50%; */
```
Comment (BareaMartin) [inline src/Products/components/ProductCard/ProductCard.module.css:10]: We shouldn't leave comments on master branch, i'll refer to this comment as DC, meaning delete comments on the entire file, if there are any.

File: src/Products/components/ProductCard/index.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import styles from './ProductCard.module.css';
+/**
+ * @param {{ product: { _id: string; name: string; price: number; provider: { _id: string, name: string } }, onRemoveProduct: (productId: string) => boolean }}
```
Comment (BareaMartin) [inline src/Products/components/ProductCard/index.js:5]: there's no need for type declarations

File: src/Products/store/reducer.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import { useSelector } from 'react-redux';
+import { DISMISS_LOADING, SET_LOADING, SET_PRODUCT, SET_PRODUCTS, SET_PROVIDERS } from './types';
+
+/**
```
Comment (BareaMartin) [inline src/Products/store/reducer.js:4]: DC and remove type declarations

File: src/Products/store/thunks.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    dispatch(dismissLoading());
+    dispatch(setProducts(prod.data));
+    return;
+  } else {
+    dispatch(dismissLoading());
+    return;
+  }
+};
+
+/**
```
Comment (BareaMartin) [inline src/Products/store/thunks.js:19]: DC

File: src/shared/components/Layout/Footer/Footer.module.css [hunk 5]
```diff
@@ -0,0 +1,33 @@
+.footer {
+  background-color: var(--color-fifth);
+  display: flex;
+  /* flex-direction: column; */
```
Comment (BareaMartin) [inline src/shared/components/Layout/Footer/Footer.module.css:4]: DC

File: src/shared/components/Layout/Header/Header.module.css [hunk 6]
```diff
@@ -0,0 +1,23 @@
+ul {
+  display: flex;
+  flex-direction: row;
+  /* width: 200px; */
```
Comment (BareaMartin) [inline src/shared/components/Layout/Header/Header.module.css:4]: DC

File: src/shared/components/LoadingSpinner/index.js [hunk 7]
```diff
@@ -0,0 +1,14 @@
+import styles from './LoadingSpinner.module.css';
+
+const LoadingSpinner = ({ size = 'normal' }) => {
+  return (
+    <div className={`${styles['lds-ring']} ${size === 'small' ? styles['ls-small'] : ''}`}>
+      <div></div>
```
Comment (BareaMartin) [inline src/shared/components/LoadingSpinner/index.js:6]: what's the point of this empty boxes?

File: src/shared/pages/Product/product.module.css [hunk 8]
```diff
@@ -0,0 +1,23 @@
+.form{
+    /* background-color: grey; */
```
Comment (BareaMartin) [inline src/shared/pages/Product/product.module.css:2]: DC

**Review by BareaMartin — APPROVED** [review]
Dejo el aprobado solamente por no haber cambios de lógica, resuelvan comentarios antes del merge porfavor.

File: src/shared/components/LoadingSpinner/index.js (hunk 7 above)
Comment (brquispe) [reply src/shared/components/LoadingSpinner/index.js:6]: Los estilos de CSS de la animación de spinner pide que siga esa estructura.

**Pull request merged**